lang/java/reef-examples/src/main/java/org/apache/reef/examples/suspend/SuspendDriver.java

```diff
@@ -295,6 +296,9 @@ public void onNext(final byte[] message) {
             try {
               suspendedTask.getActiveContext().submitTask(TaskConfiguration.CONF
                     .set(TaskConfiguration.IDENTIFIER, taskId)
+                    .set(TaskConfiguration.TASK, SuspendTestTask.class)
```

Comment:
When driver resume the task, the driver doesn't set TaskConfiguration.TASK.  So I just add this configuration.